clients/java/zts/core/src/main/java/com/yahoo/athenz/zts/ZTSClient.java

```diff
@@ -327,6 +334,23 @@ public static void lookupZTSUrl() {
             }
         }
     }
+
+    public static void initZtsFileUtil() {
+        ztsFileUtil = new ZTSFileUtil(getAccessTokenSignKeyResolver());
+        ztsFileUtil.preload();
+    }
+
+    public static void setAccessTokenSignKeyResolver(JwtsSigningKeyResolver jwtsSigningKeyResolver) {
+        resolver = jwtsSigningKeyResolver;
+    }
+
+    public static JwtsSigningKeyResolver getAccessTokenSignKeyResolver() {
```

Comment:
based on the previous comment for the setAccessTokenSignKeyResolver I don't think we need this function at all.

Reply:
Having setAccessTokenSignKeyResolver and getAccessTokenSignKeyResolver is for setting mock resolver in unit test

Reply:
the unit tests can only be done with the set method . no need for this get method.